Lombiq.Tests.UI/OrchardCoreUITestBase.cs

```diff
@@ -13,49 +13,40 @@ namespace Lombiq.Tests.UI;
 
 public abstract class OrchardCoreUITestBase
 {
+    private const string AppFolder = "AppFolder";
```

Comment:
Is there a reason this isn't using `nameof`?

Reply:
Only that it used to be a method-local constant named `appFolder`. After moving it to the top of the file it became this - and I didn't catch it.